crates/turbopack-tests/README.md

```diff
@@ -14,7 +14,11 @@ cargo nextest run -E 'test(snapshot)'
 The filter supports any substring, and only test names which contain
 that substring will run.
 
-## Updating Snapshot
+## Adding Snapshot Tests
+
+You might receive a `rust-analyzer` error upon trying to add new snapshot tests related to [turbo/shim](https://github.com/vercel/turbo/blob/main/shim/README.md). Please follow the build requirements to fix this issue. After that, adding files within the glob in [build.rs](https://github.com/vercel/turbo/blob/main/crates/turbopack-tests/build.rs) should trigger a rebuild when running the snapshot tests.
```

Comment:
The problem is probably not only related to turbopack-tests. This note should be placed somewhere more global.

Reply:
Good shout, I've reverted the changes for this file and adjusted `CONTRIBUTING.md`